romana-install/vagrant.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
```

Comment:
Since we explicitly create VMs with specific IP addresses, why do we need a Vagrant inventory script?